backend/oasst_backend/user_repository.py

```diff
@@ -62,3 +62,32 @@ def get_user_leaderboard(self, role: str) -> LeaderboardStats:
         ]
 
         return LeaderboardStats(leaderboard=result)
+
+    def query_users(
+        self,
+        limit: Optional[int] = 20,
+        ge: Optional[str] = None,
+        lt: Optional[str] = None,
+        auth_method: Optional[str] = None,
+    ) -> list[User]:
+
+        users = self.db.query(User)
+
+        if not self.api_client.trusted:
+            users = users.filter(User.api_client_id == self.api_client.id)
+
+        if auth_method:
+            users = users.filter(User.auth_method == auth_method)
+
+        if ge:
+            users = users.filter(User.display_name >= ge)
+
+        if lt:
+            users = users.filter(User.display_name < lt)
+
+        users = users.order_by(User.display_name)
```

Comment:
I know the query planner will probably take care of this, but could we do the sorting before the lt and gt comparisons?

Reply:
fixed